src/service/api.tsx

```diff
@@ -124,6 +128,50 @@ const api = {
     return obj.data.results;
   },
 
+  async getSearchFileredList(dataFilter:
+    {
+      from: number,
+      to: number,
+      gener: string | null,
+      page: number
+    } = {
+      from: 1900,
+      to: 2020,
+      gener: null,
+      page: 1,
+    }
+  ) {
+
+    function setFilteredData() {
+      const { from, to } = dataFilter
+      if (from < to) {
+        return `&primary_release_date.gte=${from}-01-01&primary_release_date.lte=${to}-01-01`
+      } else if (from > to) {
+        return `&primary_release_date.gte=${to}-01-01&primary_release_date.lte=${from}-01-01`
+      }
```

Comment:
Is this some kind of fool-protection?

Reply:
Yep)

Reply:
I believe it is bad UX, do a validation and return error, so it can be presented on UI or filter date value will be automatically changes to match from <= to